src/asteroid_tracker/asteroid_tracker.py

```diff
@@ -13,50 +16,122 @@
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
-
+format_Error = False
 
 API_URL = 'https://api.nasa.gov/neo/rest/v1/feed?start_date=%s&end_date=%s&api_key=DEMO_KEY'
 
 
 def main():
     """
-    Main function which parses input and calculates statistics.
+    Main function which takes input arguments and calls relevant functions.
     """
 
     start_date = None
     end_date = None
 
     # TODO: Task 1 - Gather user input
+    start_date, end_date = input(" ").split()
+    # start_date = '2021-01-01'
+    # end_date = 'fff'
 
-    stats = calculate_statistics(start_date, end_date)
 
+    # start_date
+    # year, month, day = map(int, start_date.split('-'))
+    # start_date = datetime.date(year, month, day)
+
+    # start_date = start_date.strip()
+
+    # end_date
+    # year, month, day = map(int, end_date.split('-'))
+    # end_date = datetime.date(year, month, day)
+
+    # end_date = end_date.strip()
+
+    stats = calculate_statistics(start_date, end_date)
     print_statistics(stats)
 
 
 def calculate_statistics(start_date, end_date):
     """
     Make an API request and calculate statistics.
     """
-
     # TODO: Task 2 - Prepare and make the HTTP request
+    global format_Error
```

Comment:
I don't understand the use of the global here? The variable is only ever used in a single function so there is no point in making it global. Also, using the `global` keyword is generally bad practice and not necessary in Python.

Instead of these `try ... except` clauses you should have checked for an HTTP error e.g.

```python
data = json.loads(response.content)

if 'http_error' in data:
    return {
        'error': {
            'code': data['code'],
            'type': data['http_error'],
            'message': data['error_message'],
        }
    }
```